modules/ROOT/pages/api-console.adoc

```diff
@@ -181,3 +181,12 @@ To enable users to select an application to populate the `clientId` and `clientS
           type: string
           (credentialType): secret
 ----
+
+If your site has a custom request access implementation, to update the application selector, dispatch this event:
+
+[source,text]
+----
+this.dispatchEvent(new Event('acm:application_created_success', {bubbles: true}))
+----
+
+If your site has a custom request access implementation and does not dispatch the event, each time a user adds an application, the user must refresh the page to show the application in the *Application* menu.
```

Comment:
Could we add this in the line 185?? somthing like:
If your site has a custom request access implementation, to update the application selector, dispatch this event if not you will have refresh the page to show the application in the *Application* menu  each time a user adds an application

Reply:
That is better! Updated.